### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by Andersson007**
Update docs

### CODE REVIEW

**Comment by Andersson007** [comment]
ready for review

**Comment by Andersson007** [comment]
cc @felixfontein

File: contribution_to_project.rst [hunk 1]
```diff
@@ -70,10 +72,10 @@ Examples:
 
 .. _Contributing-to-community:
 
-Contributing to the community
+Contributing to the Community
```
Comment (felixfontein) [inline contribution_to_project.rst:75]: Shouldn't this stay lower-case. Also in line 78.

Comment (Andersson007) [reply contribution_to_project.rst:75]: maybe:D, OK, I'll change this

Comment (Andersson007) [reply contribution_to_project.rst]: ah, yeah, it definitely should, thanks, fixed:)

**Comment by Andersson007** [comment]
@felixfontein thanks for reviewing!

**Pull request merged**